generator/lib/src/class_element_ex.dart

```diff
@@ -161,6 +161,9 @@ extension ClassElementEx on ClassElement {
             todo: 'Remove the @PrimaryKey annotation from the field or set the model type to a value different from ObjectType.embeddedObject.');
       }
 
+      // Computed fields go last. This is important for the schema generation.
+      mappedFields.sort((a, b) => a.isComputed ^ b.isComputed ? (a.isComputed ? 1 : -1) : -1);
```

Comment:
That's a bit too much math. Isn't easier to use `sortedby`? Like, on line 152, we could do something like:

```dart
      final mappedFields = fields.realmInfo.sortedBy<num>((f) => f.isComputed ? 0 : 1);
```

Reply:
We would need to pull in `import 'package:collection/collection.dart` and I dislike the need to add `<num>` bit to `sortedBy` (`<int>` won't do).

Also `List.sort` is in-place, but apparently not stable 🤔  ... nor is `sortedBy`